sync-version/action.yaml

```diff
@@ -21,6 +24,29 @@ runs:
       with:
         token: ${{ inputs.token }}
 
+    - name: Find the snapcraft.yaml path
+      id: yaml-path
+      shell: bash
+      run: |
+        if [[ -n "${{ inputs.snapcraft-yaml-path }}"; then
+          yaml_path="${{ inputs.snapcraft-yaml-path }}"
+        else
+          snapcraft_yaml_paths=(
+          "snap/snapcraft.yaml"
+          "snapcraft.yaml"
+          "build-aux/snap/snapcraft.yaml"
+          ".snapcraft.yaml"
+          )
+          
+          for file in "${snapcraft_yaml_paths[@]}"; do
+              if [[ -f "$file" ]]; then
+                  yaml_path="$file"
+              fi
+          done
+        fi
+        echo "yaml_path=${yaml_path}" >> "$GITHUB_OUTPUT"
```

Comment:
Do we want to run a check just before this to ensure there is a valid path, and exit with an error if not?

Something like:

```bash
if [[ ! -n "${yaml_path}" ]]; then
    echo "No snapcraft.yaml found" >2
    exit 1
fi
```

Reply:
https://github.com/snapcrafters/ci/pull/8/files#diff-51b17582a77e6318708af6a9aced55b4022ad76f635a34871624092ac0ca10ceR70

wrote one, but I thought it's not necessary.. as you removed it